Marlin/src/HAL/STM32/tft/tft_fsmc.cpp

```diff
@@ -100,11 +100,11 @@ void TFT_FSMC::Init() {
 
   HAL_SRAM_Init(&SRAMx, &Timing, &ExtTiming);
 
-  __HAL_RCC_DMA2_CLK_ENABLE();
-
   #ifdef STM32F1xx
-    DMAtx.Instance = DMA2_Channel1;
+    __HAL_RCC_DMA1_CLK_ENABLE();
+    DMAtx.Instance = DMA1_Channel1;
```

Comment:
you left DMA1 stuff here

Reply:
It is STM32 HAL (non-maple), there is no conflict with ADC there.

Reply:
ok